app/helpers/users_helper.rb

```diff
@@ -1,2 +1,7 @@
 module UsersHelper
+  def gravatar_for(user)
+    gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
```

Comment:
__[rubocop]__
* Do not use `::` for method calls. [:ref](https://github.com/bbatsov/ruby-style-guide#double-colons)